app/views/posts/index.html.erb

```diff
@@ -1,2 +1,34 @@
-<h1>Posts#index</h1>
-<p>Here is a list of posts for a given user</p>
+<div class="flex flex-col w-full">
+<div class="flex items-center justify-between flex-col w-full p-6 bg-white rounded-lg md:flex-row">
+
+  <div class="w-20 mr-2">
+  <%= image_tag 'img2.jpg', alt: 'icon'%>
+  </div>
+
+
+  <div class="flex justify-between w-full">
+    <%# <h2 class="flex text-bold text-2xl"><%= @post.author %></h2> %>
+    <%# <div>Number of Posts: <%= @post.comments_counter%> </div> %>
```

Comment:
- [ ]Please do not leave unused code as a comment in your codebase. this will make your code much noisier, It is always a good practice to keep your code clean from such comments. 😉 👍🏻